4.5 release page (#1138)
godotengine/godot-website: collections/_release_4_5/entry-general-editor-drop-preload-resources-as-uid.md

---
type: entry
section: general
subsection: editor
rank: 1
importance: 4
anchor: drop-preload-resources-as-uid

> title: Drop preload Resource as UID

Reviewer: I don't really understand this title, in particular the "drop".

Author: See https://github.com/godotengine/godot/pull/99094

Reviewer: OK thanks, so it's drop as in "drag and drop". I thought it might have meant drop as in "drop this feature we previously supported".

This could do with a more straightforward title, but one doesn't immediately come to mind...

Author: I used finally "Drag and drop `Resource`s in scripts to preload by UID instead of by path"

blockquote: TODO
text: |

>   With [Godot 4.4](https://godotengine.org/releases/4.4/#universal-uid-support), we introduced UIDs in order to prevent broken paths.

Reviewer 2: This is misleading, we introduced UIDs for 4.0, 4.4 introduced them for all resource types, should probably say "we extended UID support" or similar

Author: I updated the sentence and the next paragraph.

>   With [Godot 4.4](https://godotengine.org/releases/4.4/#universal-uid-support), we extended UIDs support to resources in order to prevent broken paths.

Reviewer 2: ```suggestion
  With [Godot 4.4](https://godotengine.org/releases/4.4/#universal-uid-support), we extended UID support to resources in order to prevent broken paths.
```

>   Scripts can now take advantage of this, as it is now possible to preload resources by their UID.

Reviewer 3: ```suggestion
  Scripts can now take advantage of this by preloading resources via their UID.
```
Or a similar wording – the "as it is now possible" feels a little verbose to me

>   By using UIDs instead of paths for preloading, your scripts will be more resilient, wherever the resources you're preloading are in your project.

Reviewer 2: ```suggestion
  By using UIDs instead of paths for preloading, your scripts will be more resilient, wherever the resources you’re preloading are in your project.
```
